infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/workflows/HealNodeWorkflow.java

```diff
@@ -42,34 +42,11 @@ public String getName() {
 
     @Override
     public List<Action> getActions() {
-        return Arrays.asList(new ResetNode(),
```

Comment:
I haven't changed that behaviour. I agree it should be static instead of creating new Action objects every time. Can I address that in a separate PR as it is orthogonal to this issue?

Reply:
You might as well, you already touching the code.